src/graphlog.rs

```diff
@@ -127,10 +126,9 @@ mod sapling {
 }
 
 pub fn get_graphlog<'a, K: Clone + Eq + Hash + 'a>(
-    #[allow(unused)] settings: &UserSettings,
+    settings: &UserSettings,
     formatter: &'a mut dyn Write,
 ) -> Box<dyn GraphLog<K> + 'a> {
-    #[cfg(feature = "sapling")]
     {
```

Comment:
Drop the now unnecessary braces?

Also, the default case could now be moved into the `match` expression. Then the `return`'s could be removed.

Reply:
Done. I also dropped the `mod sapling`.

Reply:
Please take another look because quite a lot changed.

By the way, what do you think about renaming `ui.graph.format` to `ui.graph.style`? I think of it more as a style than a format.

Reply:
SGTM. I didn't spend a long time thinking about the name of the option.